src/main/resources/application.yml

```diff
@@ -9,6 +9,10 @@ spring:
       database: iexec
       host: ${MONGO_HOST:localhost}
       port: ${MONGO_PORT:13202}
+      auto-index-creation: true
```

Comment:
Maybe add a note about `Automatic index creation disabled by default as of Spring Data MongoDB 3.x.`

Reply:
Good idea, that's added!